spec/ripple/conflict/resolver_spec.rb

```diff
@@ -8,10 +8,10 @@ module Conflict
       end
 
       describe 'calling the lambda returned by .to_proc' do
-        let(:sibling_1) { stub(:data => { '_type' => 'User' } ) }
-        let(:sibling_2) { stub(:data => { '_type' => 'User' } ) }
+        let(:sibling_1) { stub(:data => { '_type' => 'User' }, :content_type => 'application/json') }
```

Comment:
Agreed - making this change makes me just as nervous 😨.

I can add a `respond_to?` check to the rejection block before checking the content type. Since the stubs don't respond to `content_type`, we'll never attempt the nil check. The existing specs should still pass then.